content/post/setup-kubernetes-raspberry-pi-cluster.md

```diff
@@ -1,45 +1,45 @@
 +++
 Categories = ["Docker", "Kubernetes","K8s", "Clustering", "Cluster","Raspberry Pi", "Cloud Computing"]
 Tags = ["Docker", "Kubernetes","K8s", "Clustering", "Cluster","Raspberry Pi", "Cloud Computing"]
-date = "2016-12-26T18:03:34+01:00"
+date = "2016-12-31T12:03:34+01:00"
 more_link = "yes"
-title = "Setup of Kubernetes on a Raspberry Pi Cluster the official way"
+title = "Setup Kubernetes on a Raspberry Pi Cluster easily the official way!"
 +++
 
-Kubernetes (K8s) shares the pole position with Docker in the category "orchestration solutions for Raspberry Pi cluster". However it's setup process has always been elaborate. Now, with it's recent release, Kubernetes changed this game completely and can be up and running within no time.
+[Kubernetes](kubernetes.io) shares the pole position with Docker in the category "orchestration solutions for Raspberry Pi cluster". 
+However it's setup process has been elaborate, until [v1.4 with the kubeadm announcement](http://blog.kubernetes.io/2016/09/how-we-made-kubernetes-easy-to-install.html). 
+With that effort, Kubernetes changed this game completely and can be up and running officially within no time.
 
-I am very happy to announce that this blog post has been written in collaboration with Lucas Käldström, an independent maintainer of Kubernetes. He contributed most in order to bring Kubernetes to the ARM architecture.
+I am very happy to announce that this blog post has been written in collaboration with [Lucas Käldström](https://github.com/luxas), an independent maintainer of Kubernetes. 
+You can read his story here, in a [CNCF blogpost](https://www.cncf.io/blog/2016/11/29/diversity-scholarship-series-programming-journey-becoming-kubernetes-maintainer).
 
 
 ![SwarmClusterHA](/images/kubernetes-setup-cluster/raspberry-pi-cluster.png)
 
 
-<!--more-->
```

Comment:
This line is needed for the blog engine. Without this line, the blog post will cover the complete website, which is not intended :)